bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -243,7 +243,14 @@ def setup_run_environment(self, env):
     exit -1
 fi
 
-if [ -n "{nrnivmodlcore_call}" ]; then
+COMPILE_ONLY_NEURON=0
+
+if [[ "$2" == "--only-neuron" ]]; then
```

Comment:
If we're getting into argument parsing, I think this whole script may be re-written in another language with better CLI support (in the _future_) :)

Reply:
It's specially cool that this is inside a `.py` file 🤦 😆

Reply:
That's a valid point, I agree. For the moment we can probably get away with this since it's only one CLI argument :innocent:

Reply:
Looking at the script... that could be included in the neurodamus repository, and only a small configuration file generated by Spack, too.